graylog2-server/src/main/java/org/graylog2/search/SearchQueryParser.java

```diff
@@ -259,6 +260,8 @@ FieldValue createFieldValue(SearchQueryField field, String quotedStringValue, bo
                 return new FieldValue(Integer.parseInt(pair.getLeft()), pair.getRight(), negate);
             case LONG:
                 return new FieldValue(Long.parseLong(pair.getLeft()), pair.getRight(), negate);
+            case ID:
```

Comment:
Can you please rename this to `OBJECT_ID` to make it a bit more explicit? :slightly_smiling_face: Thank you!